src/material-experimental/mdc-slider/slider.scss

```diff
@@ -0,0 +1,34 @@
+@import '@material/slider/mixins';
+@import '../mdc-helpers/mdc-helpers';
+
+$mat-slider-min-size: 128px !default;
+$mat-slider-horizontal-margin: 8px !default;
+
+@include mdc-slider-core-styles($query: $mat-base-styles-without-animation-query);
+
+.mat-mdc-slider {
```

Comment:
Add comment that explains why these styles are necessary on top of MDC

(in general we want to have comments that explain every non-MDC style so that we can keep track of what we're adding on top and why)